packages/addons/service/oscam/package.mk

```diff
@@ -2,10 +2,10 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="oscam"
-PKG_VERSION="eafedca0d96364ca8e1f035cf30afa0f3fa9186b"
-PKG_SHA256="f73130a0ae1a2b8d31cd3f5d2709f04936724fb8d4a69e0c4c1a59f69b107be3"
-PKG_VERSION_NUMBER="11741"
-PKG_REV="0"
+PKG_VERSION="1.20"
+PKG_SHA256="97f708467db18ce4785c77db8d0c41e4eb444b116264fc2e9e8a8e4c3056f699"
```

Comment:
oscam has a new site with a much more up to date version
https://git.streamboard.tv/common/oscam
download 
`https://git.streamboard.tv/common/oscam/-/archive/${PKG_VERSION}/oscam-${PKG_VERSION}.tar.gz`

Reply:
- added in #9060